src/DynamoCore/Linting/LinterManager.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Dynamo.Core;
+using Dynamo.Extensions;
+using Dynamo.Graph.Nodes;
+using Dynamo.Graph.Workspaces;
+using Dynamo.Linting.Interfaces;
+using Dynamo.Linting.Rules;
+using Dynamo.Models;
+
+namespace Dynamo.Linting
+{
+    public class LinterManager : NotificationObject, IDisposable
+    {
+        #region Private fields
+        private readonly IExtensionManager extensionManager;
+        private LinterExtensionDescriptor activeLinter;
```

Comment:
We said we are going to serialize the active linter here right ?

Reply:
We discussed about this internally and plan is to have another entry in the dynamo file : 
`Linting : { activeLinter: adadasdds, issuesCount: .... }`